adal/devApps/XFormsApp/SecondPage.cs

```diff
@@ -79,6 +83,12 @@ public SecondPage()
                 AutomationId = "acquireTokenSilent"
             };
 
+            var acquireTokenWithPromptBehaviorAlways = new Button
```

Comment:
>acquireTokenWithPromptBehaviorAlways [](start = 16, length = 36)

This will not scale with all the prompt behaviours :) Is it too complex to have a drop down and read from that?

Reply:
discussed off-line...the promptBehaviors are tied to the specific platforms, so I could make a dropdown, but it will interfere with whatever was passed in during app creation (ex. the Activity in Android). Can make a task for this for further investigation and refactoring.

---
In reply to: [228564485](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1356#discussion_r228564485) [](ancestors = 228564485)